src/app.ts

```diff
@@ -64,29 +68,50 @@ async function app() {
       'exit with error if this is not the latest version of SUSHI',
       false
     )
-    .option('-i, --init', 'initialize a SUSHI project')
-    .option('-u, --update-dependencies', 'update FHIR packages in project configuration')
-    .version(getVersion(), '-v, --version', 'print SUSHI version')
+    .action(async function (projectPath, options) {
+      await runBuild(projectPath, options);
+    })
     .on('--help', () => {
       console.log('');
       console.log('Additional information:');
       console.log('  [path-to-fsh-project]');
       console.log('    Default: "."');
       console.log('  -o, --out <out>');
       console.log('    Default: "fsh-generated"');
+    });
+
+  program
+    .command('updateDependencies [path-to-fsh-project]')
+    .description('update FHIR packages in project configuration')
+    .action(async function (projectPath) {
+      const input = ensureInputDir(projectPath);
+      const config: Configuration = readConfig(input);
+      await updateExternalDependencies(config);
+      process.exit(0);
     })
-    .arguments('[path-to-fsh-defs]')
-    .action(function (pathToFshDefs) {
-      input = pathToFshDefs;
-    })
-    .showHelpAfterError()
-    .parse(process.argv)
-    .opts();
+    .on('--help', () => {
+      console.log('');
+      console.log('Additional information:');
+      console.log('  [path-to-fsh-project]');
+      console.log('    Default: "."');
+    });
 
-  if (program.init) {
-    await init();
-    process.exit(0);
+  program
+    .command('init')
+    .description('initialize a SUSHI project')
+    .action(async function () {
+      await init();
+      process.exit(0);
+    });
+
+  // Maintain backwards compatability with prior SUSHI command structure by defaulting to build
+  if (process.argv.length >= 2 && !SUPPORTED_COMMANDS.includes(process.argv[2])) {
+    process.argv = [...process.argv.slice(0, 2), 'build', ...process.argv.slice(2)];
```

Comment:
I think this might simplify this line. I had to think about what was happening with the two spread operators on the `slice`s.

```suggestion
    process.argv = process.argv.splice(2, 0, 'build');
```

Reply:
Fixed in [fe15249](https://github.com/FHIR/sushi/pull/1163/commits/fe15249a27e11d26faf9284e0d4a048f55960078).